helpers/dbload.ts

```diff
@@ -7,6 +7,7 @@ import { ChallengeTypes, Challenge } from './models/Challenge'
 import { AlertTypes, Alert } from './models/Alert'
 import { StarTypes, Star } from './models/Star'
 
+/* istanbul ignore next */
```

Comment:
I added jest ignore for this function because simple ternary operator doesn't warrant separate test IMO. And testing it will be rather ugly with a lot of mocks because Sequelize config isn't easily accessible.